<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scan_UI</class>
 <widget class="QWidget" name="Scan_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ImageView" name="Image_View"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Scan control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="Move_groupbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="title">
             <string>Move</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="X_move_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>150.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="Y_move_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>150.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>speed (V/s)</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QDoubleSpinBox" name="speed_move_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="minimum">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>10.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="2" rowspan="2">
                <widget class="QPushButton" name="start_move_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Move to</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="Image_groupbox">
            <property name="title">
             <string>Image</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="7" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Scan direction</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Starting corner</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="range_scan_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>150.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QSpinBox" name="readdelay_scan_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Pixel number</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Line delay (ms)</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="Ycenter_scan_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QComboBox" name="corner_scan_combobox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>upper left</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>upper right</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>lower left</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>lower right</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QComboBox" name="dir_scan_combobox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>X fast Y slow</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Y fast X slow</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>X center (V)</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Read delay (ms)</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QSpinBox" name="linedelay_scan_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="Xcenter_scan_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="pixelnum_scan_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>4096</number>
                 </property>
                 <property name="value">
                  <number>128</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Scan range (V)</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Y center  (V)</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Angle (degree)</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="angle_scan_spinbox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>180.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="start_scan_button">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>48</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="setup_button">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Setup up</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
